.github/workflows/build.yml

```diff
@@ -124,7 +124,7 @@ jobs:
 
   release:
     timeout-minutes: 15
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Wouldn't it be enough to pin it only here for the `release` job?

Reply:
I think it's better to keep a consistent version. Moreover, the release phase benefits from saved artifacts from earlier phases, see https://github.com/oyvindberg/bleep/pull/236/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R118-R123

Reply:
Exactly, let's keep it consistent! :)